requirements_all.txt

```diff
@@ -306,10 +306,12 @@ blockchain==1.4.4
 # homeassistant.components.bom
 bomradarloop==0.1.3
 
-# homeassistant.components.amazon_polly
 # homeassistant.components.route53
 boto3==1.9.233
 
+# homeassistant.components.amazon_polly
+boto3==1.9.252
```

Comment:
`boto3` has to be the same version for all integrations.

Reply:
good catch. fixed.